chaincfg/regnetparams.go

```diff
@@ -295,6 +295,34 @@ func RegNetParams() *Params {
 				StartTime:  0,             // Always available for vote
 				ExpireTime: math.MaxInt64, // Never expires
 			}},
+			10: {{
```

Comment:
Note: I have only added the deployment to regnet to enable creating the unit test in the next PR but didn't introduce it to mainnet and testnet. 

I'm not sure whether I should do this here (in this PR) or if this should be left for a future PR, since we have additional agendas that will be voted under the same deployment version.

Reply:
Absence is counted as being active, so it will have to be added to mainnet and testnet as well.